conversation/backend/src/main/java/org/entcore/conversation/controllers/ApiController.java

```diff
@@ -166,6 +171,46 @@ private Future<JsonObject> deleteFolders(final HttpServerRequest request, final
 		return getAuthenticatedUserInfos(eb, request)
 			.compose( user -> conversationService.deleteFoldersAndTrashMessages(folderIds, user) );
 	}
+
+	@Get("api/purge/list")
+	@SecuredAction(value = "", type = ActionType.RESOURCE)
+	@ResourceFilter(SuperAdminFilter.class)
+	public void purge_list(final HttpServerRequest request) {
```

Comment:
```suggestion
	public void purgeList(final HttpServerRequest request) {
```
Pour respecter les règles de nommage Java (vs Python)